deployments/deployment/nginx-plus-ingress.yaml

```diff
@@ -65,6 +65,7 @@ spec:
           - -nginx-plus
           - -nginx-configmaps=$(POD_NAMESPACE)/nginx-config
           - -default-server-tls-secret=$(POD_NAMESPACE)/default-server-secret
+         #- -include-year
```

Comment:
can specify the same in `deployments/daemon-set/nginx-plus-ingress.yaml` as well

Reply:
Good catch. I will also add it to the OSS version of the deployments